mappings/remediation_advice/remediation_advice.json

```diff
@@ -1182,22 +1182,329 @@
       ]
     },
     {
-      "id": "broken_cryptography",
-      "children": [
+    "id": "cryptographic_weakness",
+    "children": [
+        {
+        "id": "insufficient_entropy",
+        "references": [
+        ],
+        "children": [
+            {
+            "id": "limited_rng_entropy_source",
+            "remediation_advice": "1. When using the native entropy sources in a Linux environment, be sure to use /dev/random instead of /dev/urandom.  This is because, in the event of depletion of the kernel entropy pool, /dev/random blocks until the level of entropy in the system entropy pool is high enough to ensure high entropy random number generation.  In contrast, /dev/urandom does not block and will return low-entropy random numbers in the event of depletion of the system entropy pool.\n\n2. In Windows environments, ensure all elements of the Windows entropy pool exhibit appropriately high levels of randomness.\n\n3. In applications that require large volumes of high-entropy random numbers to be generated, consider using alternative RNGs, such as Intel's Digital Random Number Generator (DRNG).",
+            "references": [
+                "https://www.redhat.com/en/blog/understanding-random-number-generators-and-their-limitations-linux#:~:text=A%20source%20of%20entropy%20(RNG)&text=Random%20number%20generators%20or%20RNGS,unpredictable%20numbers%20as%20its%20output.",
+                "https://blog.cloudflare.com/ensuring-randomness-with-linuxs-random-number-generator/",
+                "https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-90B.pdf",
+                "file:///Users/nerdwell/Downloads/Whitepaper%20-%20The%20Windows%2010%20random%20number%20generation%20infrastructure.pdf"
```

Comment:
@nerdwell can you please clean this path up?

Reply:
Ah!  Good catch, @vortexau!  I've replaced that local file path with an Internet-accessible URL to the download for the referenced white paper.